README.md

```diff
@@ -27,24 +27,38 @@ are convertible to `bitcoin::PublicKey`
 completing an unsigned `bitcoin::TxIn` with appropriate data
 * Determining the specific keys, hash preimages and timelocks used to spend
 coins in a given Bitcoin transaction
-* `no_std` support enabled by disabling the `default-features` and enabling
-`"no-std"`. See `embedded/` for an example.
+* `no_std` support enabled by disabling the `default-features`. See `embedded/` for an example.
 
 More information can be found in [the documentation](https://docs.rs/miniscript)
 or in [the `examples/` directory](https://github.com/rust-bitcoin/rust-miniscript/tree/master/examples)
 
 ## Building
 
-The cargo feature `std` is enabled by default. At least one of the features `std` or `no-std` or both must be enabled.
+The cargo feature `std` is enabled by default. To enable `no_std` support you must disable default features.
 
-Enabling the `no-std` feature does not disable `std`. To disable the `std` feature you must disable default features. The `no-std` feature only enables additional features required for this crate to be usable without `std`. Both can be enabled without conflict.
+The library can be built and tested using [`cargo`](https://github.com/rust-lang/cargo/):
+
+```
+git clone git@github.com:rust-bitcoin/rust-miniscript.git
+cd rust-miniscript
+cargo build
+```
+
+You can run tests with:
+
+```
+cargo test
+```
+
+Please refer to the [`cargo` documentation](https://doc.rust-lang.org/stable/cargo/) for more
+detailed instructions.
 
 ## Minimum Supported Rust Version (MSRV)
 
 This library should always compile with any combination of features on **Rust 1.63.0**.
 
 Some dependencies do not play nicely with our MSRV, if you are running the tests
-you may need to pin some dependencies. See `./contrib/test.sh` for current pinning.
+you may need to pin some dependencies. See `./contrib/pin.sh` for current pinning.
 
```

Comment:
This change is good, its an unrelated change for this PR though. (Sorry for being picky, we just uncovered a bug in `rust-bitcoin` that slipped through review so I'm trying to review more ~anally~ diligently.

Reply:
Thanks, no worries about being thorough. It is unrelated, but I figured it was okay since it’s just an outdated path in the README.